css/style.css

```diff
@@ -15,5 +26,177 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+/* header */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2rem 5rem;
+}
+.logo {
+  display: block;
+  min-width: 2rem;
+  min-height: 2rem;
+}
+
+/* navigation */
+
+.navlist {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  list-style: none;
+}
+
+.navitems {
+  display: flex;
+  padding: 0 1.5rem;
+}
+
+.navlinks {
+  color: var(--grey);
+  text-decoration: none;
+}
+
+.navlinks:hover {
```

Comment:
Great 👍
This will change colour for mouse users who hover.
Which pseudo-class could you add to also change the colour for a keyboard user who has tabbed onto the link?